src/components/containers/Main.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import PropType from 'prop-types';
+import UrlInputForm from '../form/UrlInputForm';
+import Method from '../form/Method';
+
+export default class Main extends Component {
+
+  state = {
+    urlInput: '',
+    method: 'get'
+  };
+
+  handleUrlChange = ({ target }) => {
+    this.setState({ urlInput: target.value });
+  }
+
+  handleMethodChange = ({ target }) => {
+    this.setState({ methodInput: target.value });
+  }
+
+  render() {
+    return (
+      <>
+        <form>
+          <UrlInputForm handleUrlChange={this.handleUrlChange} />
+          <div id="methods">
+            
+            <label for="get"><Method id={'get'} handleMethodChange={this.handleMethodChange} checked="checked" />GET</label>
```

Comment:
labels can't have a "for" property, so this is creating a console error

> webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:530 Warning: Invalid DOM property `for`. Did you mean `htmlFor`?
    in label (created by Main)
    in div (created by Main)
    in form (created by Main)
    in Main (created by App)
    in section (created by App)
    in main (created by App)
    in App